runtime/onert/core/src/compiler/ExecutorFactory.h

```diff
@@ -38,6 +38,7 @@ class ExecutorFactory
 
 public:
   exec::IExecutor *create(std::unique_ptr<compiler::LoweredGraph> lowered_graph,
+                          const util::TracingCtx *tracing_ctx,
```

Comment:
(beyond this PR) I wondered why `IExecutor` requires `CompilerOptions`. It seems that the only required member is `options.executor`. I think it is an option for Executor, not Compiler. : )

Reply:
`IExecutor` doesn't requires `CompilerOptions`, but `IExecutor` factory requires because it is part of compilation. Factory uses to check executor type, tracing/profiling setting.

Reply:
@hseok-oh You're right. I think `ExecutorFactory` requires only `options.executor` (string for executor name like "LINEAR", ...) We may replace `CompilerOptions` to `const string exectur_name`.